### PR TITLE
Create CODEOWNERS

### PR DESCRIPTION
**Pull request by VeronikaJadrna**
@shayand137 @Aryan22000

### CODE REVIEW

**Review by Aryan22000 — COMMENTED** [review]
review

**Review by shayand137 — COMMENTED** [review]
review

**Review by shayand137 — COMMENTED** [review]
hiiiiiiiii

File: CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1 @@
+file for codeowners task
```
Comment (Aryan22000) [inline CODEOWNERS:1]: commentnt

Comment (shayand137) [inline CODEOWNERS:1]: sos

**Review by VeronikaJadrna — COMMENTED** [review]
review

**Review by Aryan22000 — APPROVED** [review]
review

**Pull request merged**